lib/algoliasearch-rails.rb

```diff
@@ -263,13 +268,18 @@ def algolia_reindex!(batch_size = 1000, synchronous = false)
             # delete non-indexable objects
             objects = group.select { |o| !algolia_indexable?(o, options) }.map { |o| algolia_object_id_of(o, options) }
             index.delete_objects(objects)
-            # select only indexable objects  
+            # select only indexable objects
             group = group.select { |o| algolia_indexable?(o, options) }
           end
-          objects = group.map { |o| settings.get_attributes(o).merge 'objectID' => algolia_object_id_of(o, options) }
+          objects = group.map do |o|
+            attributes = settings.get_attributes(o)
```

Comment:
When using with Sequel it doesn't return a hash, but a `Sequel::Dataset` object, which makes the next 3 lines necessary - ideally this would be hidden in the `get_attributes` method.